tvision/examples/Makefile.nmk

```diff
@@ -7,21 +7,25 @@
 #In order for the makefile to work, you have to start "Visual Studio xxxx Command prompt" 
 #                                                  or "Visual Studio xxxx x64 Cross tools command prompt"
 #I'm not sure about pre 2005 settings. Maybe you need to use them for 2005 as well, please report
-CFLAGS=-O2 -I../winnt/include -I../include -D_WIN32 /GR- /EHs-c-
 
+!ifdef WIN64
```

Comment:
See previous comment about making this makefile compatible with both 32-bit and 64-bit builds.  The makefile defaults to a 32-bit build. For a 64-bit build, the develop simply appends WIN64=1 to the command line.